DataLayer/Wearable/src/main/java/com/example/android/wearable/datalayer/MainApp.kt

```diff
@@ -46,99 +49,124 @@ import com.google.android.horologist.compose.layout.ScreenScaffold
 import com.google.android.horologist.compose.layout.rememberResponsiveColumnState
 import com.google.android.horologist.compose.material.Chip
 
-@OptIn(ExperimentalHorologistApi::class)
 @Composable
 fun MainApp(
     events: List<Event>,
     image: Bitmap?,
-    onQueryOtherDevicesClicked: () -> Unit,
-    onQueryMobileCameraClicked: () -> Unit
+    nodesViewModel: NodesViewModel
 ) {
     AppScaffold {
-        val columnState = rememberResponsiveColumnState(
-            contentPadding = ScalingLazyColumnDefaults.padding(
-                first = ItemType.Chip,
-                last = ItemType.Text
-            )
+        val navController = rememberSwipeDismissableNavController()
+        SwipeDismissableNavHost(navController = navController, startDestination = "main") {
+            composable("main") {
+                MainScreen(
+                    onShowNodesList = { navController.navigate("nodeslist") },
+                    onShowCameraNodesList = { navController.navigate("cameraNodeslist") },
+                    image,
+                    events
+                )
+            }
+            composable("nodeslist") {
+                ConnectedNodesScreen(nodesViewModel)
+            }
+            composable("cameraNodeslist") {
+                CameraNodesScreen(nodesViewModel)
+            }
+        }
+    }
+}
+
+@OptIn(ExperimentalHorologistApi::class)
+@Composable
+fun MainScreen(
+    onShowNodesList: () -> Unit,
+    onShowCameraNodesList: () -> Unit,
+    image: Bitmap?,
```

Comment:
Bitmap might not be a good type on a Composable function.  We should probably introduce some data class and mark it Stable?